controllers/progresses.js

```diff
@@ -112,8 +113,9 @@ const getProgress = async (req, res) => {
         message: error.message,
       });
     }
-    return res.status(400).json({
-      message: error.message,
+    logger.error(error.message);
+    return res.status(500).json({
+      message: INTERNAL_SERVER_ERROR_MESSAGE,
     });
   }
```

Comment:
why have we written  this twice
logger.error(error.message);
    return res.status(500).json({
      message: INTERNAL_SERVER_ERROR_MESSAGE,
    });
  }

Reply:
logger will log it in our logs. while the `res.status(500).json({`will send the reponse.

Reply:
you this answer gives me the meaning of the logger,
my doubt was we have mentioned twice the  line in the code why was that needed

Reply:
I am unable to get you. which line has been mentioned twice. can you tell the line no's which you think are repeated.

Reply:
As discussed in the call @Pratiyushkumar  I am resolving this.